src/transformers/models/llama/configuration_llama.py

```diff
@@ -158,35 +148,16 @@ def __init__(
         self.use_cache = use_cache
         self.rope_theta = rope_theta
         self.rope_scaling = rope_scaling
-        self._rope_scaling_validation()
         self.attention_bias = attention_bias
         self.attention_dropout = attention_dropout
         self.mlp_bias = mlp_bias
 
+        rope_config_validation(self)
```

Comment:
I know it wasn't there before but in the spirit of clarity maybe a small comment would be worthwhile